appserver/admingui/payara-console-extras/src/main/resources/hazelcast/hazelcast.jsf

```diff
@@ -54,7 +54,7 @@
     <!beforeCreate
         gf.restRequest(endpoint="#{sessionScope.REST_URL}/get-hazelcast-configuration"  method="GET" result="#{requestScope.resp}");
         setPageSessionAttribute(key="valueMap", value="#{requestScope.resp.data.extraProperties.getHazelcastConfiguration}");
-        setPageSessionAttribute(key="convertToFalseList", value={"enabled","dynamic"});
+        setPageSessionAttribute(key="convertToFalseList", value={"enabled","dynamic","autoIncrementPort","hostAwarePartitioning"});
```

Comment:
Note the addition of `"hostAwarePartitioning"` to the list which fixed a defect where the unchecked checkbox would not change the state of the configuration.